commands/translate.js

```diff
@@ -11,15 +8,5 @@ exports.handler = (argv) => {
     console.error('Please provide a word to translate');
   }
```

Comment:
I believe if the key is not found right now, it will show the error but will continue to call the `translateToGaro` method. I think should return back here no?